media/css/firefox/whatsnew/whatsnew-111-en.scss

```diff
@@ -0,0 +1,107 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import 'includes/dark-mode';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+
+@keyframes loading {
+    0% {
+        background-position: 100% 50%;
+    }
+
+    100% {
+        background-position: 0 50%;
+    }
+}
+
+.wnp-loading,
+.wnp-editor,
+.wnp-default {
+    &.show {
+        display: block;
+    }
+
+    &.hide {
+        display: none;
+    }
+}
+
+.wnp-content-main {
+    text-align: center;
+}
+
+.wnp-main-title {
+    margin: 0 auto $layout-md;
+    color: $color-black;
+}
+
+.wnp-main-tagline {
+    @include text-body-lg;
+    color: $color-black;
+}
+
+// Show a loading indicator until we can detect default browser
+.wnp-loading {
+    animation: loading 1.2s ease-in-out infinite;
+    background: linear-gradient(100deg, $color-light-gray-10 30%, $color-light-gray-20 50%, $color-light-gray-10 70%);
+    background-size: 400%;
+    border-radius: $border-radius-sm;
+    height: 4em;
+    margin: $layout-lg auto;
+
+    &.hide {
+        display: none;
+    }
+}
+
+// Assuming JS is available, show no content until we can detect default browser
+.js {
+    .wnp-loading,
+    .wnp-default,
+    .wnp-editor {
+        display: none;
+
+        &.show {
+            display: block;
+        }
+    }
+}
+
+// Fall back to the editor content if JS fails
+.no-js {
+    .wnp-loading,
+    .wnp-default {
+        display: none;
+    }
+
+    .wnp-editor {
+        display: block;
+    }
+}
+
+.wnp-sign-off {
+    @include text-body-sm;
+    grid-column: 1 / span 2;
+    grid-row: 5;
```

Comment:
Just realized this grid stuff got copied over from 110 but since the container isn't `display: grid` it has no effect anyway, just cruft I should have cleaned up. Oops!